src/connectionimpl.cpp

```diff
@@ -123,7 +123,9 @@ uint64_t ConnectionImpl::parse(const Buffer &buffer)
         try
         {
             // try to recognize the frame
-            ReceivedFrame receivedFrame(ReducedBuffer(buffer, processed), _maxFrame);
```

Comment:
The _full expression containing the call_ is [here](https://github.com/CopernicaMarketingSoftware/AMQP-CPP/blob/f3343ad1cb64f5a58849fcf0ee3cc22dece2393c/src/connectionimpl.cpp#L126-150).
Look at example in the [C++ Standard](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2013/n3797.pdf) on page 254.